json_rails_logger.gemspec

```diff
@@ -1,16 +1,25 @@
 # frozen_string_literal: true
 
-require './lib/json_rails_logger/version'
```

Comment:
This is an unnecessary change. I don't want to introduce Npm's `package.json` for the sole purpose of carrying version information that we already have an established pattern for.